README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 5.0.0
```

Comment:
The code changes look good. The version has been updated from 5.0.0 to 5.2.0. LGTM